scope.md

```diff
@@ -20,46 +20,58 @@ capabilities could help streamline the user experience:
 
 SPC is a tool for this last step.
 
-## Unique Features of SPC
+## Benefits and Unique Features of SPC
 
-* **Streamlined UX**. FIDO authentication provides a very good user
-experience. Our expectation is that the SPC "happy path" provides an
-even more streamlined experience. One important goal is to remove the
-need to redirect an enrolled user during a payment
+### Benefits
+
+* **Faster Authentication**. FIDO authentication provides a very good
+user experience. Our expectation is that the SPC "happy path" further
+accelerates strong customer authentication compared to other
+approaches such as one-time passwords. An important goal is to remove
+the need to redirect an enrolled user during a payment
 transaction. Another important goal is to speed up authentication
```

Comment:
The description text seems to focus more the acceleration and enablement of SCA and preventing a controlled experience. For me the title (Faster Authentication) is inaccurate. Fido is already just one click. 
The challenge with it is that there is no secure display and no payment specific fields. The industry's ideal (Frictionless using Risk) has no challenge. 
What it does do is:
* Avoid the party affecting the payment to have to redirect to a third party to authenticate. So it may save a redirect. 
* Make it closer to regulatory compliance (easier to comply) since it includes a display and signature.

What if we rather make this "Controlled Authentication" or something, which would fit the description better.